### ISSUE TITLE
Accessibility: Current day in calendar not marked with aria-current="date"

### ISSUE THREAD

**Issue by georgwittberger-t-systems-mms** [nathanreyes/v-calendar#1179]
Current behavior:

When positioning a screen reader on the current day in the calendar pane it does not announce that it is the current day.

Expected behavior:

When positioning a screen reader on the current day in the calendar pane it announces it as the current day.

Further notes:

Expected behavior can be achieved by setting ARIA attribute `aria-current="date"` on the button representing the current day. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

This has already been proposed in #346 but it is important that this attribute must be set on the current date, not the selected one.

**Comment by phalkunp** [comment]
+1 on this request.
